en/docs/manage/api-rate-limiting.md

```diff
@@ -0,0 +1,43 @@
+# API Rate Limiting
+
+API rate limiting is a technique that allows you to control the rate of requests made to an API. Rate limiting helps 
+prevent system overload and enhances API performance. When you limit the number of requests that can be made in a 
+specific time frame, you can ensure that your API is available and responsive to all users while protecting it from 
+malicious attacks.
+
+This page walks you through the steps to enable rate limiting for your APIs via Choreo and also provides information on 
+the rate-limiting options supported by Choreo.
+
+## Enable rate limiting for an API
+
+To enable rate limiting for an API, follow the steps given below:
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/). 
+2. In the **Component Listing** pane, click on the API component for which you want to apply rate limiting.
+3. In the left navigation menu, click **Manage** and then click **Settings**.
+4. In the **Settings** pane, go to the **API Settings** section and click **Edit**.
+5. Click the **Resources** tab and select a **Rate Limiting Level** depending on your requirement.
+
+## API-level rate limiting
+
+API-level rate limiting applies the allocated request count for the specified time unit to all operations in the API.
+
+![API-level rate limiting](../assets/img/manage/api-level-rate-limit.png){.cInlineImage-full}
+
+## Operation-level rate limiting
```

Comment:
Shall we explain how rate limiting works in Choreo? For e.g. a scenario and mention the slippage etc.